compiler/src/iree/compiler/Codegen/SPIRV/SPIRVVectorizeLoadStore.cpp

```diff
@@ -73,6 +75,11 @@ static unsigned calculateMemRefVectorNumBits(
     SmallVectorImpl<Operation *> &uses) {
   unsigned minBits = kMaxVectorNumBits;
   for (Operation *op : uses) {
+    if (isa<gpu::SubgroupMmaLoadMatrixOp, gpu::SubgroupMmaStoreMatrixOp>(op)) {
+      // GPU subgroup MMA ops do not care about the memref element type. It can
+      // load from whatever memref element type.
+      continue;
```

Comment:
one case we would not be able to change is if the stride is not divisible by the new memref type size. I believe it is possible so would be good to check for it to be sure we don't miscompile.

Reply:
Yeah good catch! Sorry this was sort of done in a hurry..